webapp/src/sui.tsx

```diff
@@ -90,55 +91,87 @@ export class UiElement<T extends WithPopupProps> extends data.Component<T, {}> {
 }
 
 export class DropdownMenuItem extends UiElement<DropdownProps> {
```

Comment:
The idea of all those changes in the DropdownMenuItem is to be give the focus to the selected element of the menu without closing the dropdown (because by default, it close if we give the focus to an item of the menu).